services/queue/src/exchanges.js

```diff
@@ -353,7 +353,7 @@ exchanges.declare({
     'be submitted against an already resolved task group.',
   ].join('\n'),
   routingKey: buildTaskGroupRoutingKey(),
-  schema: 'task-group-resolved.json#',
+  schema: 'task-group-changed-message.json#',
```

Comment:
@djmitche do you remember if there were any practices for pulse message schema changes? 

I see hard-coded `version: 1` field, so I don't think anyone was ever using different versions of the same message. 

What happens here is that we let several events share one schema, and that schema would include extra fields, which were not defined in the previous one.

I'm marking this as a major release because of this to at least warn whoever was using clients for the previous version

Reply:
Much like `/v1/` in the API paths, I think `version: 1` was added without much thought to how it might be used in the future.

In general, though, adding to APIs isn't an incompatible change, so no special compatibility work is required.  We've added fields to the github service's messages a few times, iirc.

Marking it as a major change probably still makes sense, though, for exactly the reason you stated.

Reply:
Right, thanks! 

I checked rabbitMQ and the only client listening to that exchange is actually a github service. 
And it seems like payload validation is on the publisher side, not the consumer.